### PR TITLE
Fix docker user issue

### PR DESCRIPTION
**Pull request by avimehenwal**
docker would create results dir with root permission.
we are using FS mount to copy files in container
https://github.com/CN-UPB/tng-sdk-benchmark/wiki/Bug

https://github.com/CN-UPB/tng-sdk-benchmark/wiki/Bug

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{
```
Comment (qarawlus) [inline .vscode/launch.json:1]: Remove this file: Add it to .gitignore

File: Dockerfile [hunk 2]
```diff
@@ -40,8 +40,11 @@ RUN pip install tavern  # fix (try to remove once the tavern installation is not
 COPY . /tng-sdk-benchmark
 WORKDIR /tng-sdk-benchmark
 RUN python setup.py install
-
-#
-# Runtime
-#
+ARG USER_NAME=avi
```
Comment (qarawlus) [inline Dockerfile:43]: try to remove your username from here

File: examples/osm_ped/cirros_2vnf_ns/cirros_2vnf_nsd.yaml [hunk 3]
```diff
@@ -0,0 +1,27 @@
+nsd:nsd-catalog:
```
Comment (qarawlus) [inline examples/osm_ped/cirros_2vnf_ns/cirros_2vnf_nsd.yaml:1]: Is this a ped? check https://github.com/CN-UPB/tng-sdk-benchmark/blob/master/examples/peds/ped_calibration_stress.yml

File: examples/osm_ped/cirros_vnf/cirros_vnfd.yaml [hunk 4]
```diff
@@ -0,0 +1,48 @@
+vnfd:vnfd-catalog:
```
Comment (qarawlus) [inline examples/osm_ped/cirros_vnf/cirros_vnfd.yaml:1]: This is also not a ped. Put these files in the correct positions.

**Review by qarawlus — COMMENTED** [review]
Good job, but we need better coordination :)

**Comment by avimehenwal** [comment]
Closing this PR as it needs cleaning up .... It mixes multiple concerns

**Pull request closed without merging**